docs/transpile/custom_passes.mdx

```diff
@@ -0,0 +1,101 @@
+---
+title: Transpilation passes beyond the standard set
+description: Passes not included in the transpiler default optimization levels.
+
+---
+# Custom passes 
```

Comment:
Is it correct to call this section "Custom Passes"? To me "custom passes" implies you're creating the passes yourself, but really you are taking existing passes from qiskit but putting them together in a custom passmanager. So I would have thought a better title for this would be "custom passmanagers". But I admit this section is a bit beyond my expertise so if you think "custom passes" is the correct terminology we can stick with that

Reply:
I renamed it to "Customizing transpilation"